### PR TITLE
[ti_eclecticiq][Enhancement] Enable SSL configuration

### PR DESCRIPTION
**Pull request by petarc-eiq**
### Enhancement: Enable SSL configuration

Motivation: some users use EIQ IC platform with invalid SSL certificates and don't intend to renew or add a valid certificate. Still want to use that instance at least for testing purpose and we need to enable them.

Implementation: added "SSL Configuration" yaml field to allow users to configure outgoing requests SSL certificate validation. This way they have full control over this part of the communication and they can turn off SSL validation by setting `verification_mode` to `none`.

### Bugfix: parsing incoming data fields that contain dots

Bug: conditions for dot expanders are bad and they prevent reading input data properly. This causes data loss.

Fix: the conditions are updated (fixed) to enable parsing incoming data fields that contain dots.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] changelog.yml is updated according to standards

## How to test this PR locally

Scenario: SSL configuration field works as desired.
Given: updated integration is installed in an Elastic Stack environment
And: EIQ IC is running with on https with invalid certificate
And: integration is set up to use that IC instance with `verification_mode=none`
When: integration agent runs
Then: communication is successful.

Scenario: dot expanders are set up properly.
Given: updated integration is installed in an Elastic Stack environment
And: observables that are added to dataset to be downloaded contain populated fields that should be "dot expanded" (source.names, meta.tags, meta.ingest_time, meta.estimated_threat_start_time, meta.estimated_threat_end_time, meta.estimated_observed_time)
When: integration agent runs
Then: values from "dot expanded" fields are mapped into ECS fields (event.provider, tags, event.created, event.start, event.end, threat.indicator.first_seen).

### CODE REVIEW

File: packages/ti_eclecticiq/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,12 @@
 # newer versions go on top
+- version: "0.1.1"
```
Comment (bhapas) [inline packages/ti_eclecticiq/changelog.yml:2]: As there is an enhancement can you pleaase make it `0.2.0`

Comment (petarc-eiq) [reply packages/ti_eclecticiq/changelog.yml]: Done, thanks for the comment.

**Review by bhapas — APPROVED** [review]

**Comment by zed-eiq** [comment]
Saw the PR was blocked by conflicts with `main`; resolved them: https://github.com/elastic/integrations/pull/8911/commits/5146ed24325e341c63cfd5194d2952ca37b3a91c

**Pull request merged**